### ISSUE TITLE
[A11Y] [iOS] Sidebar Menu and Close Button are not labelled appropriately.

### ISSUE THREAD

**Issue by spotukuchi** [ampproject/ampstart#383]
**ENVIRONMENT:**
Form factor : iPAD
Model : MC705LL/A
Version : 9.3.5(13G36)
Browser : Safari 

**Pre-requisite:**
Turn 'VoiceOver' ON. ( Settings->General->Accessibility->VoiceOver->ON )

**STEPS TO REPRODUCE:**
1. Navigate to the URL : https://ampstart.com
2. Try to linear navigate to Sidebar Menu and content within it.

**ACTUAL RESULTS:**
1. When user navigates to the Menu button, VoiceOver reads some random label i.e, 'try gram per heaven button' for the Sidebar Menu.
2. Close button is labelled as 'Button'.

**EXPECTED RESULTS:**
1. Sidebar Menu button should be labelled appropriately.
2.Close button should have an appropriate label.

Please see the attached screenshots for reference.

![close button](https://cloud.githubusercontent.com/assets/27025792/24883470/e7a91496-1df9-11e7-8ca5-78388fe342f8.PNG)
![sidebar menu](https://cloud.githubusercontent.com/assets/27025792/24883471/e7bba912-1df9-11e7-9e92-5f35d939f2ee.PNG)

**Comment by torch2424** [comment]
Fixed in my a11y bugs branch

[Commit Fix](https://github.com/torch2424/ampstart/commit/eeb8d43395d09cb39e26ac860625acbbcd310507)
